charts/blaze/values.yaml

```
@@ -33,6 +33,39 @@ terminologyService:
 # -- sets the value for the `JAVA_TOOL_OPTIONS` environment variable.
 javaToolOptions: "-XX:-OmitStackTraceInFastThrow"
 
+frontend:
+  enabled: false
+  replicaCount: 1
+  # -- annotations to set on the frontend deployment's pod
+  podAnnotations: {}
+  image:
+    registry: ghcr.io
+    repository: samply/blaze-frontend
+    tag: 0.34.0@sha256:d56e1793e17ba95ef63e02e3ce52146ef6a341b3314aa59cc6d79301c455976c
+    pullPolicy: IfNotPresent
+  imagePullSecrets: []
+  # -- configure the resource requests and limits for the frontend pod
+  resources: {}
+  # -- environment variables for configuring the frontend pod
+  env: []
+  service:
+    # -- type of service the frontend uses
+    type: ClusterIP
+    # -- the port for the main endpoint
+    port: 3000
+  # -- for traefik specifically, a traefik ingressRoute can be created and parameterized with this helm chart
+  # -- other ingress controllers would have to make use of additional ingresses and annotations and/or their own CRDs
+  traefik:
```

```suggestion
  ingressRoute:
```

nur als vorschlag - vielleicht etwas eindeutiger.